src/parser/jobs/sam/modules/AoeChecker.js

```diff
@@ -0,0 +1,136 @@
+	_badKenkiCasts = 0 //amount of Kenki Casts that didn't break the damage gain threshold
+
+	constructor(...args) {
+		super(...args)
+		this.addHook('aoedamage', {by: 'player'}, this._onAoe)
+		this.addHook('complete', this._onComplete)
+	}
+
+	_onAoe(event) {
+
+		const action = event.ability.guid
+
+		//Step 1: Find break point
+
+		if (GAIN_AT_2.has(action)) {
+
+			//Step 2: Check break point
+
+			if (event.hits.length < 2) {
+				//Step 3: Check type of resource used and increment
+
+				if (AOE_GCDS.has(action)) {
+
+					this._badAoeGCDs++
+				} else if (AOE_SENS.has(action)) {
+
+					this._badSenCasts++
+				} else if (AOE_KENKI.has(action)) {
+
+					this._badKenkiCasts++
+				}
+			}
+
+		} else if (GAIN_AT_3.has(action)) {
+
+			//Step 2: Check break point
+
+			if (event.hits.length < 3) {
+				//Step 3: Check type of resource used and increment
+
+				if (AOE_GCDS.has(action)) {
+
+					this._badAoeGCDs++
+				} else if (AOE_SENS.has(action)) {
+
+					this._badSenCasts++
+				} else if (AOE_KENKI.has(action)) {
+
+					this._badKenkiCasts++
+				}
+			}
+		}
+	}
+
+	_onComplete() {
+
+		//SUGGESTION TIME!
+
+		this.suggestions.add(new TieredSuggestion({
+			icon: ACTIONS.HISSATSU_GUREN.icon,
+			content: <Trans id= "sam.aoechecker.suggestions.badkenki.content">
+			Using aoe moves when you don't reach the target threshold for it to be a gain is a loss.
+			</Trans>,
+			tiers: {
+				1: SEVERITY.MINOR,
+				3: SEVERITY.MEDIUM,
+				5: SEVERITY.MAJOR,
+			},
+			value: this._badKenkiCasts,
+			why: <Trans id = "sam.aoechecker.suggestions.aoekenki.why">
+					You misused kenki moves <Plural value ={this._badKenkiCasts} one="# time" other="# times" /> during the fight.
+			</Trans>,
+		}))
+
+		this.suggestions.add(new TieredSuggestion({
+			icon: ACTIONS.TENKA_GOKEN.icon,
+			content: <Trans id= "sam.aoechecker.suggestions.badsen.content">
+                        Using aoe moves when you don't reach the target threshold for it to be a gain is a loss.
+			</Trans>,
+			tiers: {
+				1: SEVERITY.MINOR,
+				3: SEVERITY.MEDIUM,
+				5: SEVERITY.MAJOR,
+			},
+			value: this._badSenCasts,
+			why: <Trans id = "sam.aoechecker.suggestions.badsen.why">
+                                        You misused sen moves <Plural value ={this._badSenCasts} one="# time" other="# times" /> during the fight.
+			</Trans>,
+		}))
+
+		this.suggestions.add(new TieredSuggestion({
+			icon: ACTIONS.HISSATSU_GUREN.icon,
+			content: <Trans id= "sam.aoechecker.suggestions.badgcd.content">
+                        Using aoe moves when you don't reach the target threshold for it to be a gain is a loss.
+			</Trans>,
+			tiers: {
+				1: SEVERITY.MINOR,
+				3: SEVERITY.MEDIUM,
+				5: SEVERITY.MAJOR,
+			},
+			value: this._badAoeGCDs,
+			why: <Trans id = "sam.aoechecker.suggestions.badgcd.why">
+                                        You misused kenki moves <Plural value ={this._badAoeGCDs} one="# time" other="# times" /> during the fight.
```

Comment:
Seems like the wording here isn't updated to reflect that this one is triggered by bad GCDs.